src/assets/i18n/ar.json

```diff
@@ -109,5 +110,45 @@
   "translate_upload_file_placeholder": "اسحب وافلت أي الصور او مستندات قد تكون مفيدة في شرح موجزك هنا (الحد الاقصي لحجم الملف:25 مبجابايت)",
   "translate_enter_skills_here_placeholder": "إذا كان طلبك لا يحتاج لمختصين في مجال معين ننصحك بتوجيه طلبك لمجال أي شغل. لدينا نخبة من المشتغلين الجاهزون لمساعدتك في اي اختصاص",
   "translate_submit": "إرسال",
-  "translate_back": "رجوع"
+  "translate_back": "رجوع",
+  "translate_shogol_platform": "منصة شغل",
+  "translate_shogol_platform_descrption": "هناك حقيقة مثبتة منذ زمن طويل وهي أن المحتوى المقروء لصفحة ما سيلهي القارئ عن التركيز على الشكل الخارجي للنص أو شكل توضع الفقرات في الصفحة التي يقرأها",
+  "translate_about_platform": "عن المنصة",
+  "translate_new_concept": "مفهوم جديد",
+  "translate_freedom_of_action": "حرية في التعامل",
+  "translate_appreciation_of_time": "تثمين للوقت",
+  "translate_inclusive_services": "شمولية في الخدمات",
+  "translate_why_order_a_better_price": "لماذا طلب عرض سعر أفضل؟",
+  "translate_info": "معلومات",
+  "translate_why_order_a_price": "لماذا طلب عرض السعر ..؟",
+  "translate_why_order_a_price_first": "أولاً : ستطلب ولن تبحث وستوفر عنا البحث",
+  "translate_why_order_a_price_second": "ثانياً : ستكشف سعر السوق للخدمة التى تبحث عنها",
+  "translate_why_order_a_price_third": "ثالثاً : ستتصفح السيرة الذكية للمشتغلين الجاهزين لخدمتك",
+  "translate_why_order_a_price_fourth": "رابعاً : ستختار السعر و المشتغل الانسب لك بكل ثقه وراحه بال",
+  "translate_describe_your_project": "صف مشروعك هنا",
+  "translate_next": "التالي",
+  "translate_reveal_market_pricese": "تكشف أسعار السوق",
+  "translate_saving_time": "توفير الوقت",
+  "translate_all_you_have_to_do": "كل اللي عليك!",
+  "translate_all_you_have_to_do_worker": "كل اللي عليك كونك مشتغل",
+  "translate_all_you_have_to_do_user": "كل اللي عليك كونك مستخدم",
+  "translate_discover_yourself": "أكتشف نفسك",
+  "translate_highlights_of_your_skills": "أبرز مهاراتك",
+  "translate_receive_your_requests": "استقبل طلباتك",
+  "translate_receive_your_money": "استلم فلوسك ",
+  "translate_request_an_worker": "أطلب مشتغل",
+  "translate_reveal_market_price": "اكشف سعر السوق",
+  "translate_choose_whats_suitable": "اختر المناسب لك",
+  "translate_close_your_order": "أغلق طلبك",
+  "translate_some_services_and_posts_on_incumbency": "بعض الخدمات والوظائف على شغل",
+  "translate_most_important_professional_services": "أهم الخدمات الاحترافية لتطوير وتنمية أعمالك",
+  "translate_web_desining": "تصميم مواقع , جافاسكربت",
+  "translate_web_application_specialists": "متخصصو تطبيقات الويب التقدمية",
+  "translate_web_application_specialists_descrption": "مصمم جرافيك مرحبا انا أبو عدنان السوري اعمل لدي شركة فرانكو للادوات الصحية و لدي العديد من الخبرات في هذا المجال",
+  "translate_price_250_SAR": "250 ريال",
+  "translate_30_minutes_ago": "قبل 30 دقيقة",
```

Comment:
This not should like that 
It should be dynamic.

for price it can be number + translated currency 
for minutes the key may be "translate_n_minutes_ago" and the number comes from pipe.